src/cascadia/WindowsTerminalUniversal/PropertySheet.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file necessary?

Reply:
It looks like it's related to the Visual Studio Properties sheet for the project file. I'm just going to leave it alone in case we need to customize something about the CppWinRT build for the Universal project.